src/main/java/com/stuypulse/robot/commands/ControlPanelSpinToColorCommand.java

```diff
@@ -1,32 +1,27 @@
 package com.stuypulse.robot.commands;
 
+import com.stuypulse.robot.Constants;
 import com.stuypulse.robot.subsystems.ControlPanel;
 import edu.wpi.first.wpilibj.DriverStation;
 import edu.wpi.first.wpilibj.util.Color;
 import edu.wpi.first.wpilibj2.command.CommandBase;
 
-/**
- * An example command that uses an example subsystem.
- */
-public class ControlPanelTurnAutoCommand extends CommandBase {
+public class ControlPanelSpinToColorCommand extends CommandBase {
     private final ControlPanel m_cPanel;
     private Color goal;
-
-    /**
-     * Creates a new ExampleCommand.
-     *
-     * @param subsystem The subsystem used by this command.
-     */
-    public ControlPanelTurnAutoCommand(ControlPanel subsystem) {
-        m_cPanel = subsystem;
-        // Use addRequirements() here to declare subsystem dependencies.
-        addRequirements(subsystem);
+    public ControlPanelSpinToColorCommand(ControlPanel cPanel) {
+        m_cPanel = cPanel;
```

Comment:
Please refrain from using the m/k convention, and stick to standard Java camel case, and other conventions